code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -18,7 +18,7 @@
 	exotic_bloodtype = "L"
 	disliked_food = GRAIN | DAIRY
 	liked_food = GROSS | MEAT
-

```

Comment:
Could you re-add this part? It looks really ugly otherwise.

Reply:
It looks like this right now:
https://github.com/tgstation/tgstation/blob/3edd031c86355ab0bbe5f0995aef96685f04d7f0/code/modules/mob/living/carbon/human/species_types/lizardpeople.dm#L20-L22

And that's really ugly looking, we always have a padding space between type definitions and procs, or even between procs, it just looks better that way.

Reply:
Resolvez